src/parser/ColumnTypeDef.h

```diff
@@ -13,8 +13,36 @@ namespace nebula {
 struct ColumnTypeDef {
     explicit ColumnTypeDef(meta::cpp2::PropertyType t, int16_t l = 0) : type(t), typeLen(l) {}
 
+    std::string toString() const {
+        switch (this->type) {
```

Comment:
You can use `SchemaUtil::typeToString` directly.

Reply:
I prefer to replace SchemaUtil::typeToString implementation with following codes.